.cicd/package.sh

```diff
@@ -2,12 +2,14 @@
 set -eo pipefail
 . ./.cicd/helpers/general.sh
 mkdir -p $BUILD_DIR
-if [[ $(uname) == 'Darwin' ]]; then
+if [[ $(uname) == 'Darwin' && $FORCE_LINUX != true ]]; then
     bash -c "cd build/packages && chmod 755 ./*.sh && ./generate_package.sh brew"
     ARTIFACT='*.rb;*.tar.gz'
     cd build/packages
     [[ -d x86_64 ]] && cd 'x86_64' # backwards-compatibility with release/1.6.x
-    buildkite-agent artifact upload "./$ARTIFACT" --agent-access-token $BUILDKITE_AGENT_ACCESS_TOKEN
+    if [[ "$BUILDKITE" == 'true' ]]; then
```

Comment:
Other script uses `BUILDKITE` to gate some buildkite features, so I used it here also. I had initially a check for the `buildkite-agent` command directly, I could revert to that if you prefer.

Reply:
👍